utils/spice_getwindata.pro

```diff
@@ -135,14 +135,18 @@
 ;       Ver.1, 3-Feb-2020, Martin Wiesmann
 ;         modified from iris_getwindata
 ;       Version 2, 8-Jul-2021, William Thompson, corrected time calculation
+;       Version 3, 25-Jan-2022, Terje Fredvik. Added keyword nodumbell, if set
+;       mask any pixels with contributions from the dumbbells, or any other
+;       pixels below or above the narrow slit. Added keyword
+;       approximated_slit, if set (when slit_only is set) use a quicker way 
+;       of estimating the pixels to be masked
 ;-
-; $Id: 2022-01-21 14:45 CET $
-
+; $Id: 2022-01-31 13:38 CET $
 
 FUNCTION spice_getwindata, input_file, input_iwin, keep_sat=keep_sat, $
   clean=clean, wrange=wrange, verbose=verbose, $
   ixrange=ixrange, normalize=normalize, quiet=quiet, $
-  calib=calib, perang=perang
+  calib=calib, perang=perang, slit_only=slit_only, approximated_slit=approximated_slit
 
```

Comment:
Eh... finnes det noe bedre navn for "perang"? Er det for sent å endre det? Vi kan vel i det minste endre det interne navnet, altså skrive "perang=forstaaelig_variabelnavn"?

Reply:
spice_getwindata må nok ryddes litt opp i, det er mye som skjer der, eller som kommentarene sier at skjer, som er IRIS-spesifikt og som blir feil for SPICE. F.eks. er intensitetsenhetene i SPICE L2-filer 'W/m2/sr/nm', og da blir kommentarene til CALIB og PERANG forvirrende.